content/docs/introduction/roadmap.md

```diff
@@ -6,16 +6,89 @@ redirectFrom:
 
-The Neon **Limited Preview** started in February, 2022, and was made available to a small number of select users and friends.
+As always, we welcome all feedback. If you see something in Neon's implementation of Postgres that you disagree with, that can be improved, or anything missing that you feel should be there, [let us know](link to best place for that).
+
+## What's new
+
+For the latest features and fixes, check our [Changelog](docs/changelog), updated every Friday. You can subscribe to updates using our [RSS feed](https://neon.tech/docs/changelog/rss.xml).
+
+## What's coming
+
+Here's a snapshot of what we're working on now:
+
+* **Branch restore with Time Travel Assist**
+
+    Instantly restore a branch to an earlier state in its history. Use Time Travel Assist to run queries against historical data, making sure you are restoring to the right point in time.
+
+* **Autoscaling visualizations**
+
+    Adding dynamic charts, populated with real-time data, to help you visualize how autoscaling impacts CPU and RAM usage over time. The first of many features to improve observability.
+
+* **Better onboarding**
+
+    Ongoing improvements to the onboarding experience are in the works. You can expect incremental updates in the upcoming weeks.
+
+* **More integrations**
+
+    We are adding new integrations all the time. We are also making ongoing improvements to the **Integrations** management page, to make it easier to add, manage, and request new integrations.
+
+    See [Manage Integrations](/docs/manage/integrations) for more details.
+
+* **Smoother email notifications**
+
+   We are improving the notification flow for our recent addition of email-based authentication, which we added in [November 2023](docs/changelog/2023-11-02-console#email-authentication-support)](docs/changelog/2023-11-02-console#email-authentication-support). 
+
+* **Improved navigation in the Neon Console**
+
+   New breadcrumbs, page headings, and other design changes are coming to the Neon Console, to help you better navigate your project and its branches.
+
+* **Easier access to support**
+
+    For Pro Tier users, we are making it easier to get access to the support form in the Neon Console, from anywhere: our website, our docs, blog posts, or anywhere you find a link to support.
+
+
+## What we're working on next
+
+And here's a glimpse at the features we'll be taking on next:
+
+* **More regions**
+
+    Support for more regions will be released in the coming months. [Is this actually coming?]
+
+* **Branching improvements**
+
+    We're adding more features to support making branching an integral part of your development workflow. This will include features in the Neon Console, but also case studies and documentation to help you make the most out of Neon's instant branching capability. Other planned improvements include:
+
+    * Showing the last active branch on the **Branches** page. This helps make it easier for you to clean up unused branches.
```

Comment:
Can we add anything else to this list with confidence? Otherwise, we can drop the bullet point and make it a single item.